calendar.php

```diff
@@ -899,7 +899,7 @@
 		{
 			$privatecheck = '';
 		}
-		$start_date = explode("-", gmdate("j-n-Y-g:i A", $event['starttime']+$event['timezone']*3600));
+		$start_date = explode("-", gmdate("j-n-Y", $event['starttime']+$event['timezone']*3600));
```

Comment:
The date string that is generated by the gmdate() at these locations is immediatly split at the '-' characters.

Of the resulting array only the first (day of month), second (month) and third (year) elements are used in the code afterwards. The last element of the array (time) isn't used anywhere, therefore I removed it in all of these locations.